src/handlers/user.rs

```diff
@@ -82,11 +87,18 @@ pub async fn get_user_without_auth(
     Ok(Json(user))
 }
 
+const MAX_BIO_LENGTH: usize = 10;
+
 pub async fn update_user_bio(
     Extension(auth_data): Extension<AuthData>,
     State(state): State<Arc<AppState>>,
     Json(bio): Json<Bio>,
 ) -> Result<Json<User>, AppError> {
+    if bio.len() > MAX_BIO_LENGTH {
+        return Err(AppError::BioTooLong(
+            "Bio exceeds maximum length".to_string(),
```

Comment:
This message string is not dynamic, you can move this to enum variant like so

```
    #[error("Bio exceeds maximum lenght")]
    BioTooLong,
```
```suggestion
        return Err(AppError::BioTooLong)
           
```